contracts/0.8.9/WithdrawalQueueBase.sol

```diff
@@ -494,7 +489,7 @@ abstract contract WithdrawalQueueBase {
     }
 
     /// @dev Calculates discounted ether value for `_requestId` using a provided `_hint`. Checks if hint is valid
-    /// @return claimableEther discounted eth for `_requestId`. Returns 0 if request is not claimable
```

Comment:
I believe that the 'returns 0...' part should be stated for `_getClaimableEther`